app/models/subscription.rb

```diff
@@ -0,0 +1,15 @@
+class Subscription < ActiveRecord::Base
+
+  belongs_to :issue
+  belongs_to :user
+
+  # ToReview: I can't get the system to save this properly, it always save 0 as the value
```

Comment:
That's because you have an incorrect schema in the database. You might need to clean your database and seed it again.